### DISCUSSION TITLE
Is it OK to install `monterey` bottle on Ventura macOS?

### DISCUSSION THREAD

**Discussion by rafrafek** [aws/aws-cli#7390] [Q&A]
Is it OK to install `monterey` bottle on Ventura macOS?

`awscli` uses `monterey` bottle when installing on Ventura macOS by using Homebrew, while `awscli` dependencies use `ventura` bottles:

<img width="533" alt="image" src="https://user-images.githubusercontent.com/23004737/199457314-9492ab50-f550-467f-822f-f76a5cb239fa.png">

**Comment by tim-finnigan** [comment]
Hi @rafrafek thanks for reaching out. The official AWS CLI installation instructions for v2 can be found here: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html. I'm not sure about the specific answer to your question but would recommend using one of the installation methods documented in the page linked above.

**Reply by rafrafek** [reply]
I can not use official guide because official guide provides package for intel based Macs. I have Apple silicon based Mac.

<img width="630" alt="image" src="https://user-images.githubusercontent.com/23004737/201303884-ac4e2c07-e655-42c2-8785-fcd8eb0d70e9.png">
